apps/backend/test/backend_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule BackendTest do
+  use ExUnit.Case
+  doctest Backend
+
+  test "greets the world" do
```

Comment:
remove this file altogether and maybe whole test/ folders if we don't have any tests.